### ISSUE TITLE
Ошибка при загрузке конфиг-файла

### ISSUE THREAD

**Issue by guru430033** [bookpauk/inpx-web#50]
При попытке прописать libDir не ключом, а в конфиг-файле происходит вот что:

Error while loading "C:\inpx-web/.inpx-web.exe/config.json": Unexpected token F in JSON at position 76

Причем, 76 - это как раз позиция символа F в строке "libDir": "B:\Flibusta.Net"

Бекслеш ошибочно считывается, как экранирующий символ? При замене его на обычный слеш все работает.

**Comment by makss** [comment]
да, `\` экранирует, т.е. в примере должно быть так:  `"libDir": "B:\\Flibusta.Net"`
быстро проверить коректность json можно онлайн, например, здесь: http://json.parser.online.fr/ 
или гугл всегда поскажет рабочие сервисы
